core/jobs.py

```diff
@@ -127,11 +127,13 @@ def _create_new_job():
         return transaction_services.run_in_transaction(_create_new_job)
 
     @classmethod
-    def enqueue(cls, job_id, additional_job_params=None):
+    def enqueue(cls, job_id, queue_name='default', additional_job_params=None):
```

Comment:
Done, although I left "default" in queue.yaml so that tests could use it as a hint that which queue is being used isn't what's getting tested.